frontend/src/pages/Afterword/index.js

```diff
@@ -82,11 +98,13 @@ class Afterword extends React.Component<Props, State> {
     const apiUrl = `${HOST_BACKEND}/api/v1/participant-scores/${participantUuid}`;
     const response = await fetch(apiUrl);
     const json = await response.json();
+    const debrief = json.debrief;
     const scores =
       json.scores && json.scores.length > 0
         ? this.generateRows(json.scores)
         : [];
     this.setState({ scores });
+    this.setState({ debrief });
```

Comment:
I dont know if we have time to this but maybe is a good opportunity to present @ruilealsilva to redux API calls. What do you thing @bendiggle ??

Reply:
@dsantoro Yeah I think it will be beneficial. Although, it will use the same action you have created in your PR. Maybe you could sit with @ruilealsilva and show him how it works? (maybe remember the return statement this time 😂)

Also @ruilealsilva for the future, there's no need to do multiple setStates after each other (this will make the component re-render multiple times. You can just do `this.setState({ scores, debrief })`

Reply:
@bendiggle  I'm so stupid haha!

Reply:
> Also @ruilealsilva for the future, there's no need to do multiple setStates after each other (this will make the component re-render multiple times. You can just do `this.setState({ scores, debrief })`

yeah... i know... 🤦 I had it separated so i could set it to true to test the button and not forget to "clean it". Ended up cleaning the true and not the duplicated setState. It's already fixed and I've already changed the styled components with ``.